includes/admin/class-amp-editor-blocks.php

```diff
@@ -43,6 +43,22 @@ public function init() {
 		if ( function_exists( 'gutenberg_init' ) ) {
 			add_action( 'enqueue_block_editor_assets', array( $this, 'enqueue_block_editor_assets' ) );
 			add_filter( 'wp_kses_allowed_html', array( $this, 'whitelist_block_atts_in_wp_kses_allowed_html' ), 10, 2 );
+
+			/*
+			 * Dirty AMP is required when a site is in native mode but not all templates are being served
```

Comment:
This comment is hard to follow; perhaps not mentioning "Dirty AMP" but explaining directly the semantics of the filter and action functions may be clearer. Just a thought.